scripts/globus/setup_collection_directory.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+# Description
+#
+# This script sets up a collection directory for DataFed.
+# It checks for necessary permissions and environment variables,
+# creates the directory if it doesn't exist, and sets the correct ownership.
+set -euf -o pipefail
+
+readonly SCRIPT=$(realpath "${BASH_SOURCE}")
+readonly SOURCE=$(dirname "$SCRIPT")
+readonly PROJECT_ROOT=$(realpath "${SOURCE}"/../../)
+readonly FILENAME=$(basename "${BASH_SOURCE}")
+
```

Comment:
This is a nice script but I don't remember seeing any admin documentation for it's use.